hil/migrations/versions/02f7e9607e16_delete_legacy_obm_support.py

```diff
@@ -0,0 +1,49 @@
+"""Delete legacy obm support
+
+Revision ID: 02f7e9607e16
+Revises: d65a9dc873d7
+Create Date: 2018-09-04 17:00:44.359952
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '02f7e9607e16'
+down_revision = 'd65a9dc873d7'
+branch_labels = ('hil',)
+
+# pylint: disable=missing-docstring
+
+
+def upgrade():
+    engine = op.get_bind()
+
+    # These tables may or may not exist, depending on what extensions the user
+    # had enabled:
+    for driver_table in 'mock_obm', 'ipmi':
+        if engine.dialect.has_table(engine, driver_table):
+            op.drop_table(driver_table)
+
+    op.drop_constraint(u'node_obm_id_fkey', 'node', type_='foreignkey')
+    op.drop_column('node', 'obm_id')
+    op.drop_table('obm')
+
+
+def downgrade():
```

Comment:
so we won't officially support downgrading but we'll just let it be here for someone that might need it?

Reply:
I thinks falls under the heading of #980; this is sortof status quo of writing the downgrade function but not really supporting it.